packages/ui/src/components/shadcn/ui/dropdown-menu.tsx

```diff
@@ -25,14 +32,14 @@ const DropdownMenuSubTrigger = React.forwardRef<
   <DropdownMenuPrimitive.SubTrigger
     ref={ref}
     className={cn(
-      'flex cursor-default select-none items-center rounded-sm px-2 py-1.5 text-sm outline-none focus:bg-overlay-hover data-[state=open]:bg-overlay-hover data-[state=open]:text-strong',
+      'flex cursor-default select-none items-center rounded-sm px-2 py-1.5 text-sm outline-none focus:bg-overlay-hover data-[state=open]:bg-overlay-hover data-[state=open]:text-strong space-x-2',
```

Comment:
this might break things if other things are added in.
it also means you can't flex-col or space-y

Reply:
Will remove it and add to all dropdown items.